.chloggen/634.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: messaging
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add gcp_pubsub unary pull example and gcp_pubsub attributes
```

It seems the changelog is still out-of-date

```suggestion
note: Add new GCP attributes: `gcp.project.id`, `messaging.gcp_pubsub.message.delivery_attempt`,
`messaging.gcp_pubsub.message.ack_deadline`,
`messaging.gcp_pubsub.message.ack_id`,
`messaging.gcp_pubsub.message.delivery_attempt` and add messaging examples
```